### DISCUSSION TITLE
When installing netcdf-c-4.9.2 , an error :"configure: error: Compiling a test with HDF5 failed"

### DISCUSSION THREAD

**Discussion by LiuJiao0408** [Unidata/netcdf-c#2950] [Q&A] [answered]
the error is "configure: error: Compiling a test with HDF5 failed.  Either hdf5.h cannot be found, or config.log should be checked for other reason."
but I have give the path of hdf5 first
"
export CPPFLAGS="-I$DIR/curl/incldue -I$DIR/ZJPlib/incldue -I$DIR/hdf5/incldue -I$DIR/mpich/incldue"
export LDFLAGS="-L$DIR/curl/lib -L$DIR/ZJPlib/lib -L$DIR/hdf5/lib -L$DIR/mpich/lib"
"

**Comment by LiuJiao0408** [comment]
[config.log](https://github.com/user-attachments/files/16286366/config.log)
this is the config.log

**Comment by LiuJiao0408** [comment, marked as answer]
export CPPFLAGS="-I$DIR/curl/incldue -I$DIR/ZJPlib/incldue -I$DIR/hdf5/incldue -I$DIR/mpich/incldue"
the “incldue” wrong

**Reply by WardF** [reply]
This appears to be correct, is this issue corrected if you change this?

**Reply by LiuJiao0408** [reply]
yes, This is a spelling error in the pathword. It was my carelessness caused it, sorry for that!
